Controller.cpp

```diff
@@ -20,7 +20,13 @@ int main()
 {
     Log("Audio controller starting");
     
-    // Begin qjackctl
+    FILE *jack = popen("qjackctl","w"); // Begin qjackctl (not yet working)
```

Comment:
After a little more reading, I think it will be better to go a different route for starting qjackctl. popen calls a couple other functions under the covers (fork, exec, and pipe). The pipe call is what allows us to easily send input to the started process. In the case of qjackctl, though, we don't actually need that functionality. And calling popen doesn't give a good way of getting the process ID of the app we're starting up (to later kill it).

Given that, I think it's probably better to call [fork](http://linux.die.net/man/2/fork) and [exec](http://linux.die.net/man/3/exec) directly so that we can easily get the PID and kill qjackctl when we're done with it. I'll see if I can get that working tonight.